clients/web/test/testUtils.js

```diff
@@ -309,7 +333,12 @@ girderTest.testMetadata = function () {
             }, 'the add metadata button to appear');
             runs(function () {
                 expectedNum = $(".g-widget-metadata-row").length;
-                $('.g-widget-metadata-add-button:visible').click();
+
+                if (type === 'json') {
+                    $('a.g-add-json-metadata').click();
+                } else {
+                    $('a.g-add-simple-metadata').click();
+                }
```

Comment:
The above blocks can be reduced to

    $('a.g-add-' + type + '-metadata').click();